src/DotNetWorker.Grpc/Definition/GrpcFunctionDefinition.cs

```diff
@@ -71,16 +76,116 @@ public GrpcFunctionDefinition(FunctionLoadRequest loadRequest, IMethodInfoLocato
         {
             // Get the input converter attribute information, if present on the parameter.
             var inputConverterAttribute = parameterInfo?.GetCustomAttribute<InputConverterAttribute>();
-
             if (inputConverterAttribute != null)
             {
                 return new Dictionary<string, object>()
                 {
                     { PropertyBagKeys.ConverterType, inputConverterAttribute.ConverterType.AssemblyQualifiedName! }
                 }.ToImmutableDictionary();
             }
+            else
+            {
+                var inputAttribute = parameterInfo?.GetCustomAttribute<InputBindingAttribute>();
+                var triggerAttribute = parameterInfo?.GetCustomAttribute<TriggerBindingAttribute>();
+
+                return GetBindingAttributePropertiesDictionary(inputAttribute) ??
+                        GetBindingAttributePropertiesDictionary(triggerAttribute) ??
+                        ImmutableDictionary<string, object>.Empty;
+            }
+        }
+
+        private ImmutableDictionary<string, object>? GetBindingAttributePropertiesDictionary(BindingAttribute? bindingAttribute)
+        {
+            if (bindingAttribute is null)
+            {
+                return null;
+            }
+
+            var output = new Dictionary<string, object>();
+            bool isInputConverterAttributeAdvertised = false;
+
+            // ConverterTypesDictionary will be "object" part of the return value of this method - ImmutableDictionary<string, object>
+            // The dictionary has key of type IInputConverter and value as Properties of that converter (specifies supported types and support for Json Deserialization)
+            var converterTypesDictionary = new Dictionary<Type, ConverterProperties>();
+
+            IEnumerable<Attribute> customAttributes = bindingAttribute.GetType().GetCustomAttributes();
+
+            foreach (Attribute element in customAttributes)
+            {
+                if (element.GetType() == typeof(InputConverterAttribute))
+                {
+                    var attribute = element as InputConverterAttribute;
+
+                    if (attribute is not null)
+                    {
+                        isInputConverterAttributeAdvertised = true;
+                        Type converter = attribute.ConverterType;
+                        ConverterProperties supportedTypes = GetTypesSupportedByConverter(converter);
+                        converterTypesDictionary.Add(converter, supportedTypes);
+                    }
+                }
+            }
 
-            return ImmutableDictionary<string, object>.Empty;
+            output.Add(PropertyBagKeys.BindingAttributeSupportedConverters, converterTypesDictionary);
+
+            if (isInputConverterAttributeAdvertised)
+            {
+                output.Add(PropertyBagKeys.EnableFallbackConverters, false);
```

Comment:
The default should be `true` to keep logic identical with the way it is today.

Reply:
Done